src/utils/getFilteredNotes.js

```diff
@@ -21,9 +21,13 @@ export function getFilteredNotes(notes, sortByDate, sortByPriority, label){
             break;
     }
 
-    if(label.length===0){
-        return notes;
-    }
+    notes = label.length!==0 
+    ? notes.filter(note=>note.labels.some(labelItem=>label.includes(labelItem)))
+    : notes;
+    
+    notes = searchTerm
```

Comment:
Appreciation: Good that you've added search functionality to your application.

Reply:
Thanks